homeassistant/components/sensor/recorder.py

```diff
@@ -26,6 +27,7 @@
     DEVICE_CLASS_HUMIDITY: {"mean", "min", "max"},
     DEVICE_CLASS_PRESSURE: {"mean", "min", "max"},
     DEVICE_CLASS_TEMPERATURE: {"mean", "min", "max"},
+    DEVICE_CLASS_MONETARY: {"sum"},
```

Comment:
If it's a price per X or a currency conversion rate, we would want `mean`, `min` and `max`.

Reply:
Ah never mind, that would be for a `DEVICE_CLASS_PRICE`. This is for just total cost.